packages/salesforcedx-vscode-core/package.json

```diff
@@ -163,10 +163,6 @@
           "command": "sfdx.force.conflict.open",
           "when": "view == conflicts && viewItem == conflict-actions",
           "group": "inline"
-        },
-        {
-          "command": "sfdx.create.manifest",
-          "when": "view == metadata && viewItem == component"
```

Comment:
A for my education question...why do we sometimes have the `"when": "false"` and other times don't?

Reply:
@RitamAgrawal?